README.md

```diff
@@ -16,6 +16,11 @@ store in logs folder, or generate automatically with RStudio Connect.
 
 # How to validate data with assertr
 
+TODO: instalacja wersji ropensci z mastera! (trzeba uporządkować nasz fork i jego rekomendować)
```

Comment:
to be removed

Reply:
This is weird, I don't see this TODO in the recent code. This seems like a comment to some development commit with temporary changes before issuing a pull request. I suggest we should merge with squash, but please let me know if you prefer keeping the commit history and removing this TODO from the history

Reply:
My bad, I've been checking changes commit by commit that's why I saw it.